netrw/rewire/rwcm.py

```diff
@@ -0,0 +1,68 @@
+from . import BaseRewirer
+import copy
+import itertools as it
+import random
+import networkx as nx
+import numpy as np
+
+
+class rwcm(BaseRewirer):
```

Comment:
Can you add a docstring detailing what this rewiring method does (with citation, if there is one).

Also, can you give the class name a `CamelCase` descriptor? I'm not familiar with this approach so I don't know if there's a more descriptive name than `RwcmRewirer`, though.